sunpy/net/vso/vso.py

```diff
@@ -806,6 +807,19 @@ def unknown_method(self, response):
 class InteractiveVSOClient(VSOClient):
     """ Client for use in the REPL. Prompts user for data if required. """
     def multiple_choices(self, choices, response):
+        """
+        ?
+
+        Parameters
+        ----------
+            choices : ?
+
+            response : ?
+
+        Returns
+        -------
+            ? : ?
+        """
```

Comment:
Um, can we have something better than question marks here?  Even a phrase like "not yet documented" would be better than "?".